src/mmw/js/src/core/modals/views.js

```diff
@@ -388,8 +389,34 @@ var PlotView = ModalBaseView.extend({
     }
 });
 
+var CreateAccountPersuasionModalView = ModalBaseView.extend({
```

Comment:
That might be confusing now that we have an actual splash page.  Maybe `SignUpMarketingModalView`?